src/sage/schemes/curves/affine_curve.py

```diff
@@ -1781,41 +1827,55 @@ def fundamental_group(self, simplified=True, puiseux=False):
         .. NOTE::
 
             The curve must be defined over the rationals or a number field
-            with an embedding over `\QQbar`.
+            with an embedding over `\QQbar`. This functionality requires
+            the ``sirocco`` package to be installed.
```

Comment:
It didn't seem like this was needed before. If that was the case, this is a regression.

Reply:
I do not understand, it has been needed from the beginning.

Reply:
Ah, sorry, I missed that there was a `# optional - sirocco` marker for the block previously. This would be a case where it is better to have the block since nearly all tests require it (plus it would mean fewer changes).